src/tests/opertate.test.js

```diff
@@ -0,0 +1,9 @@
+import operate from '../logic/operate';
+
+const numberOne = 2;
+const numberTwo = 4;
+const operation = 'x';
+
+test('return the result of multiplication of two numbers', () => {
+  expect(operate(numberOne, numberTwo, operation)).toEqual('8');
+});
```

Comment:
- [ ] If you see the project requirement then you should test all the functionalities & scenarios of operate.js that is `+,-,*,/,%`. etc. Please write the test for all the functionality.